uni/lib/view/common_widgets/request_dependent_widget_builder.dart

```diff
@@ -39,7 +39,7 @@ class RequestDependentWidgetBuilder extends StatelessWidget {
           case RequestStatus.none:
             return contentChecker
                 ? contentGenerator(content, context)
-                : onNullContent;
+                : onNullContent ?? Container();
```

Comment:
If it goes wrong, the request will fail, which this widget handles.
Since no content is loaded onto the provider from the database, the content will be null every time the page is first loaded. To avoid the visual artefact, I just put a blank widget there. See the refactor.